contributed_definitions/NXapm.nxdl.xml

```diff
@@ -0,0 +1,1129 @@
+                        <dim index="2" value="3"/>
+                    </dimensions>
+                </field>
+                <group name="reconstruction_parameter" recommended="true" type="NXcollection">
+                    <doc>
+                         Different reconstruction protocols exist. Although these approaches
+                         are qualitatively similar, each protocol uses different parameters
+                         (and interprets these differently). The source code to IVAS/APSuite
+                         is not open. For now users should store reconstruction parameter
+                         in a collection.
+                    </doc>
+                </group>
+                <group name="naive_point_cloud_density_map" type="NXprocess">
+                    <doc>
+                         To get a first overview of the reconstructed dataset,
+                         the format conversion computes a simple 3d histogram
+                         of the ion density using one nanometer cubic bins without
+                         applying smoothening algorithms on this histogram.
+                    </doc>
+                    <field name="program" type="NX_CHAR">
+                        <doc>
+                             Given name of the program that was used to perform this computation.
+                             Apart from the classical approach to use AMETEK software for this
+                             processing step, a number of open-source alternative tools exists.
+                        </doc>
+                        <attribute name="version" type="NX_CHAR">
+                            <doc>
+                                 Program version plus build number, commit hash, or description of
+                                 an ever persistent resource where the source code of the program
+                                 and build instructions can be found so that the program can be
+                                 configured in such a manner that the result file is ideally
+                                 recreatable yielding the same results.
+                            </doc>
+                        </attribute>
+                    </field>
+                    <group type="NXdata">
+                        <doc>
+                             A default three-dimensional histogram of the total number of ions in each bin.
+                        </doc>
+                    </group>
+                </group>
+            </group>
+            <group name="ranging" recommended="true" type="NXprocess">
+                <doc>
+                     Data post-processing step in which elemental, isotopic,
+                     and/or molecular identities are assigned to the ions.
+                     The documentation of these steps is based on ideas
+                     described in the literature:
+                     * [M. K. Miller](https://doi.org/10.1002/sia.1719)
```

Comment:
needs a blank line before and after the bullet list, otherwise displays as a single paragraph

Reply:
fixed

Reply:
Done